<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabStructures</class>
 <widget class="QWidget" name="TabStructures">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="StyledComboBox" name="comboSeedSource">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Current seed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>From matching seeds list</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Seed(s):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Analyze the structures within an area.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabStructures">
      <attribute name="title">
       <string>Structures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridOptions">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioMap">
           <property name="text">
            <string>Active map selections</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioAll">
           <property name="text">
            <string>All structures</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkCollect">
           <property name="text">
            <string>Collect all instances</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeStructs">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>seed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>structure</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>z</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>details</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabQuads">
      <attribute name="title">
       <string>Quad-Structures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeQuads">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>seed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>distance</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>z</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>radius</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>spawn area</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridArea">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="layoutArea">
       <item>
        <widget class="QLabel" name="labelX1">
         <property name="toolTip">
          <string>Lower bound (inclusive)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CoordEdit" name="lineX1">
         <property name="toolTip">
          <string>Lower bound (inclusive)</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelZ1">
         <property name="toolTip">
          <string>Lower bound (inclusive)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Z&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CoordEdit" name="lineZ1">
         <property name="toolTip">
          <string>Lower bound (inclusive)</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelX2">
         <property name="toolTip">
          <string>Upper bound (inclusive)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CoordEdit" name="lineX2">
         <property name="toolTip">
          <string>Upper bound (inclusive)</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelZ2">
         <property name="toolTip">
          <string>Upper bound (inclusive)</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Z&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CoordEdit" name="lineZ2">
         <property name="toolTip">
          <string>Upper bound (inclusive)</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonFromVisible">
       <property name="text">
        <string>From visible</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordEdit</class>
   <extends>QLineEdit</extends>
   <header>src/conditiondialog.h</header>
  </customwidget>
  <customwidget>
   <class>StyledComboBox</class>
   <extends>QComboBox</extends>
   <header>src/widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
